### PR TITLE
Fix persons props table

### PR DESCRIPTION
**Pull request by paolodamico**
## Changes

As reported by a [user](https://posthog.slack.com/archives/C01F31E9PRD/p1641481159001000), the props table was displaying with very cumbersome widths, making it really hard to read.

**Before**

**After**

## How did you test this code?
Tested on the following pages:
- Person page
- Events table page
- Person (actor) modal

### CODE REVIEW

File: frontend/src/lib/components/PropertiesTable.scss [hunk 1]
```diff
@@ -1,5 +1,28 @@
...
 
+.properties-table {
+    overflow-x: auto;
+
+    table {
+        width: 100%;
+    }
+
+    tr {
+        border-bottom: 1px solid $border;
```
Comment (Twixes) [inline frontend/src/lib/components/PropertiesTable.scss:11]: This results in an orphaned "second" border at the bottom:
Screen Shot 2022-01-07 at 16 32 27

**Review by Twixes — COMMENTED** [review]
This is OK for a stopgap, but there already is a design for a more viable solution at https://github.com/PostHog/posthog/issues/7776#issuecomment-996906613, which uses the standard `LemonTable`. Instead of moving to a manually generated `table` in `PropertiesTable`, could we fix the issue within the constraints of `LemonTable`? 🤔

**Comment by paolodamico** [comment]
I ran into quite a bit of issues getting the LemonTable to work with this, but I agree that's the long-term solution. I'm not sure what your timeline is on that, should we merge this and then get back to LemonTable when that's fixed? Or do you plan to tackle that soon

**Comment by paolodamico** [comment]
Alright looks like @Twixes PR that fully refactors this is in, so closing.

**Pull request closed without merging**